about/about.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <title> Code Comp </title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" type="image/png" href="favicon.png"/>
+
+  </head>
+
+    <!--First section-->
+
+    <div style="background-image: url(./images/backpic.png);
```

Comment:
Avoid using inline CSS and instead use an `<img>` tag with `role="presentation"` to add an image following WAI-ARIA accessibility standards